### ISSUE TITLE
Output the requested colums to TSV SNV/Indel report for WGS workflow

### ISSUE THREAD

**Issue by jasonwalker80** [genome/analysis-workflows#705]
@apaul7 and @johnegarza,

We need to update the appropriate YAML inputs to translate the correct VCF fields to the final SNV/Indel TSV report.

**Comment by apaul7** [comment]
once #642 gets merged the process_inputs.pl will have the inputs set as

variants_to_table_fields:
	```
	['CHROM', 'POS', 'ID', 'REF', 'ALT'];
	```
variants_to_table_genotype_fields:
	```
	['GT', 'AD', 'DP'];
	```
vep_to_table_fields:
	```
	['Consequence', 'SYMBOL', 'Feature', 'HGVSc', 'HGVSp', 'ExACpLI', 'LoFtool', 'CADD_raw', 'REVEL_score', 'gnomADe_AF', 'Consequence', 'SYMBOL', 'Feature_type', 'Feature', 'HGVSc', 'HGVSp', 'cDNA_position', 'CDS_position', 'Protein_position', 'Amino_acids', 'Codons', 'HGNC_ID', 'Existing_variation', 'CLIN_SIG', 'PHENO', 'clinvar_CLINSIGN', 'clinvar_PHENOTYPE', 'clinvar_SCORE', 'clinvar_RCVACC', 'clinvar_TESTEDINGTR', 'clinvar_PHENOTYPELIST', 'clinvar_NUMSUBMIT', 'clinvar_GUIDELINES'];
	```

'SpliceRegion' will be added to the vep fields once #706 is resolved

**Comment by apaul7** [comment]
resolved with #642 and #718
